<launch>
  <!-- Output -->
  <arg name="input/car_signals" default="classification/classified/car/traffic_signals"/>
  <arg name="input/pedestrian_signals" default="classification/classified/pedestrian/traffic_signals"/>
  <arg name="output/traffic_light_signals" default="classification/traffic_signals"/>
  <!-- <arg name="all_camera_namespaces"/> -->
  <arg name="namespace" default="camera6"/>

  <!-- Node -->
  <group>
    <push-ros-namespace namespace="camera6"/>
    <node pkg="autoware_traffic_light_signals_merger" exec="traffic_light_signals_merger_node" name="autoware_traffic_light_signals_merger_node" output="screen">
      <remap from="input/car_signals" to="$(var input/car_signals)"/>
      <remap from="input/pedestrian_signals" to="$(var input/pedestrian_signals)"/>
      <remap from="output/traffic_light_signals" to="$(var output/traffic_light_signals)"/>
    </node>
  </group>
</launch>